tools/ansible/roles/dockerfile/templates/Dockerfile.j2

```diff
@@ -141,7 +141,7 @@ RUN dnf -y install \
     wget \
     diffutils \
     unzip && \
-    npm install -g n && n 14.15.1 && dnf remove -y nodejs
+    npm install -g n && n 14.15.1 && npm install -g npm@latest && dnf remove -y nodejs
```

Comment:
@nixocio I think we should pin this to a version, if possible.

Reply:
@jakemcdermott, suggestion accepted, tested and updated.